### PR TITLE
set fake-hwclock

### PR DESCRIPTION
**Pull request by firecyberice**
Helper for hypriot/image-builder-rpi#334

### CODE REVIEW

File: flash [hunk 1]
```diff
@@ -704,6 +703,11 @@ if [ -f "${boot}/device-init.yaml" ]; then
   fi
 fi
 
+# set fake-hwclock
+if [ -f "${boot}/fake-hwclock.data" ]; then
+  date --utc --rfc-3339=seconds |cut -d+ -f1 > "${boot}/fake-hwclock.data"
```
Comment (StefanScherer) [inline flash]: These `date` options don't not work on macOS, but the following works both on Linux and macOS.

```suggestion
  TZ=UTC date '+%Y-%m-%d %H:%M:%S' > "${boot}/fake-hwclock.data"
```

File: test/fake-hwclock.bats [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+teardown() {
+  umount_sd_boot /tmp/boot
+  rm -f $img
+  unstub_diskutil
+}
+
+@test "fake-hwclock: flash updates fake-hwclock.data" {
+  expected=$(TZ=UTC date '+%Y-%m-%d %H:%M')
```
Comment (StefanScherer) [inline test/fake-hwclock.bats]: Remove seconds from expected string here, this hopefully runs the tests stable enough in CI without a change in the minute.

**Comment by eloots** [comment]
Great! Gave it a try and it works as advertised!

**Comment by manicmonkey** [comment]
Never heard of fake-hwclock before now - very clean fix! :)

**Pull request merged**